includes/functions/functions_email.php

```diff
@@ -359,7 +359,13 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
       /**
        * Send the email. If an error occurs, trap it and display it in the messageStack
        */
-      if (!$mail->send()) {
+      $rc = 1;
+      try { 
+         $rc = $mail->send(); 
+      } catch (Exception $e) {
+         $rc = 0; 
+      }
+      if (!$rc) { 
```

Comment:
Definitely better.

How do you feel about simplifying it further?
Something like initializing to false before calling:

$success = false;
try ... $success = $mail->send()

and then catch requires no code

and then if (!$success) ...

Reply:
Great.  This is small and easy to backport.